openedx/features/course_experience/utils.py

```diff
@@ -214,6 +214,15 @@ def is_block_structure_complete_for_assignments(block_data, block_key):
     if children:
         return all(is_block_structure_complete_for_assignments(block_data, child_key) for child_key in children)
 
+    category = block_data.get_xblock_field(block_key, 'category')
+    if category in ('course', 'chapter', 'sequential', 'vertical'):
```

Comment:
Should we include verticals here? That would mark the subsection above it as incomplete. Would that be like, FBE stuff? Where the user can see the html stuff but not the actual problem sets? In that case we'd want it to be incomplete yeah?

Reply:
Just doing sequentials would fix my current bug. So it's the safer change... But I think verticals is right too?